### PR TITLE
replace jqModal with bootstrap modal

### PR DESCRIPTION
**Pull request by orangejenny**
cherry-picked from B3 app manager migration

before:
screen shot 2016-01-14 at 6 10 20 pm

after:
screen shot 2016-01-14 at 6 10 43 pm

I'd like to get rid of jqModal altogether, but it's still used in couchlog, and adding bootstrap there requires upgrading jquery (from 1.4), which causes an error in datatables. Not going to dig into that today.

@TylerSheffels

### CODE REVIEW

**Comment by czue** [comment]
feel free to just get rid of the popup functionality in couchlog and change to always open in a new window if it makes your life easier. i think it's very rarely used.

**Comment by orangejenny** [comment]
@czue that's good news! will do.

File: corehq/apps/app_manager/templates/app_manager/app_view.html [hunk 1]
```diff
@@ -358,4 +297,36 @@
...
+    </div>
+    <div class="modal-body form-horizontal">
+        <div class="alert alert-danger password-mismatch hide">
+            {% trans "The passwords don't match. Please try again." %}
+        </div>
+        <div class="alert alert-danger password-empty hide">
+            {% trans "Please enter a password or cancel." %}
+        </div>
+        <div class="control-group password">
+            <label class="control-label">Password</label>
```
Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/app_view.html]: couple missing `trans` tags down here.

**Comment by czue** [comment]
this is so much better it's almost comical. :star:

File: corehq/apps/app_manager/templates/app_manager/app_view.html [hunk 2]
```diff
@@ -128,85 +128,43 @@
...
-                popup.css({
-                    display: 'block',
-                    position: 'absolute',
-                    top: position.top,
-                    left: position.left,
-                }).addClass('ui-corner-tr ui-corner-br ui-corner-bl shadow');
-                $('body').append(popup);
-                password1.focus();
-            });
+                $popupLink = $('<a/>').attr({href: '#'}).text("Reset"),
```
Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/app_view.html]: sneaky trans in here too

**Comment by czue** [comment]
looks good to me pending a couple translations!

**Comment by orangejenny** [comment]
added translations

**Comment by snopoke** [comment]
So much better and finally getting rid of all that JS code!

**Pull request merged**